static/js/zamboni/collections.js

```diff
@@ -640,4 +640,9 @@ $(document).ready(function () {
     if ($('#details-edit').length && $('div.notification-box').length) {
         $(document).scrollTop($("div.primary").position().top);
     }
+
+    var notificationBox = document.querySelector('.notification-box');
```

Comment:
We should do this with CSS and not with JS. You should be able to add `margin-bottom: 5px` to `.notification-box` (from the look of it) to get this to work instead.